apps/functions-subscription/src/main.ts

```diff
@@ -1,5 +1,5 @@
 import { app } from '@azure/functions';
-import { InfoFunction } from './functions/info';
+import { InfoFunction } from './adapters/azure/functions/info';
```

Comment:
just a little consideration: why this structure? Do we need to classify azure/functions adapters or handlers with others?

Reply:
The idea behind is that the adapters are implementations and implementations can be done using different technologies, that's why I decided to explicit the technology used also in the folders structure.

It is also true that, at the moment, the only implementation we are going to do is the one with azure functions, so the "old" structure would be good as well.

Do you have any preferences about the structures?

Reply:
Since this is a tailored azure functions I'd rather keep "old" structure. Anyway it's not a blocking consideration :)